src/frameworks/base.js

```diff
@@ -0,0 +1,42 @@
+class FrameworkBase {
+  constructor() {}
+
+  get name() {
+    return this.constructor.name;
+  }
+
+  static isAutoLoaded() {
+    return true;
+  }
+
+  isApplicable() {
+    return true;
+  }
+
+  useConfig(someConfigs = {}) {}
+
+  supportedCapabilities() {
+    return [];
+  }
+
+  canAnswerCapability(capability) {
+    return this.supportedCapabilities().includes(capability);
+  }
+}
+
+class GDPRFramework extends FrameworkBase {
```

Comment:
_Curiosity/definitely-not-blocking_: is there a reason this isn't in its own file? (Also, should it have a `name` attribute?)

Reply:
oops, this was a left over experiment. Removed.